src/gui/advancedsettings.cpp

```diff
@@ -321,6 +346,14 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(LIBTORRENT_HEADER, QString("<b>%1</b>").arg(tr("libtorrent Section")), labelLibtorrentLink);
     static_cast<QLabel *>(cellWidget(LIBTORRENT_HEADER, PROPERTY))->setAlignment(Qt::AlignCenter | Qt::AlignVCenter);
 
+#if defined(Q_OS_WIN)
+    m_comboBoxOSMemoryPriority.addItems({tr("Normal"), tr("Below normal"), tr("Medium"), tr("Low"), tr("Very low")});
+    m_comboBoxOSMemoryPriority.setCurrentIndex(static_cast<int>(session->getOSMemoryPriority()));
```

Comment:
I wouldn't cast OSMemoryPriority to index. Just map it explicitly like you make reverse mapping above.